internal/events/types/moderator_change/moderator_change_event.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package moderator_change
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/twitchdev/twitch-cli/internal/events"
+	"github.com/twitchdev/twitch-cli/internal/models"
+	"github.com/twitchdev/twitch-cli/internal/util"
+)
+
+var transportsSupported = map[string]bool{
+	models.TransportWebSub:   true,
+	models.TransportEventSub: false,
```

Comment:
https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#channelmoderatoradd-beta <- may be worth adding beta support?

Reply:
Will open a separate PR